gradle-plugin/README.md

```diff
@@ -6,7 +6,7 @@ For usage of this plugin, see the documentation.
 
 ```
 $ cd gradle-plugin
-$ ./gradlew build install  
+$ ./gradlew build publishToMavenLocal
```

Comment:
"install" task was from old "maven" plugin. "publishToMavenLocal" is from "maven-publish" plugin.